crates/router/tests/connectors/utils.rs

```diff
@@ -273,9 +272,8 @@ pub trait ConnectorActions: Connector {
         payment_data: Option<types::RefundsData>,
         payment_info: Option<PaymentInfo>,
     ) -> Result<types::RefundSyncRouterData, Report<ConnectorError>> {
-        let max_try = 3;
-        let mut curr_try = 1;
-        while curr_try <= max_try {
+        let max_tries = 2;
+        for curr_try in 0..max_tries {
```

Comment:
FYI, this would not do 3 retries, it would stop after 0 and 1.
If you need this to be 3 retries, then either use:

`0..3` or `1..=3` (or `0..=2`, but I feel this is less readable)

If 2 retries is what you intend, let me know, I'll approve the PR.